sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -1026,10 +1026,10 @@ class CachedTableSuite extends QueryTest with SQLTestUtils
         withTable(s"$db.cachedTable") {
           // Create table 'cachedTable' in temp db for testing purpose.
           spark.catalog.createTable(
-            s"$db.cachedTable",
-            "PARQUET",
-            StructType(Array(StructField("key", StringType))),
-            Map("LOCATION" -> path.toURI.toString))
+            tableName = s"$db.cachedTable",
+            source = "PARQUET",
+            schema = StructType(Array(StructField("key", StringType))),
+            options = Map("LOCATION" -> path.toURI.toString))
 
```

Comment:
I had to make these changes so that the correct method would be invoked. I'm not fluent in Scala, but at least in Python I'd consider this style to also be better form since explicitly naming the parameters eliminates ambiguity.

Would this be considered an API breaking change, though? Since folks who are using the method with unnamed parameters will have trouble once `description` gets added in.

If so, would the solution be to move `description` to the end of the parameter list?

Reply:
> This patch fails MiMa tests.

I guess that's a yes, this breaks binary compatibility. Let me try moving the parameter around.